olittwhmcs/serializer.py

```diff
@@ -64,6 +64,33 @@ def get_client_request_parameters(email=None, client_id=None):
     return parameters
 
 
+def update_client_request_parameters(**kwargs):
+    parameters = get_default_parameters()
+    parameters.update({'action': 'UpdateClient'})
+    for param, value in kwargs.items():
+        if param == 'first_name':
```

Comment:
A better implementation would be to have a dictionary mapping of param against WHMCS parameter key
ie `{'param': 'whmcs_param'}`. In this case you would do `parameters.update({param_map[param]: value})`